### ISSUE TITLE
feature request!!

### ISSUE THREAD

**Issue by anvesh9652** [Duroktar/Wolf#80]
not working for direct functions calls 
`def hi():`
`    return 'hi hello'`

`hi()   // No output showing`

`we have to use either print(hi()) or store in a variable x = hi()`

**is there any feature that directly prints for function calling**

like this

`hi()                   'hi hello' `

**Comment by Duroktar** [comment]
Yes, I believe a ["comment macro"](https://github.com/Duroktar/Wolf#comment-macros) is what you're looking for. ie: `?` as the first character in the comment on the same line.

Example:

``` py
hi() #? 'hi hello'
```

**Comment by anvesh9652** [comment]
Thank you for your response. But isn't it great if it works directly without typing "comment macro"? Are there any consequences behind implementing without "comment macro"? Don't get me wrong.  I only started using wolf a few days ago, but I fell in love with this extension😅. I thought it would be great if we had that feature either directly or with the setting enabled. are there any chances of making it possible?

**Comment by Duroktar** [comment]
I suppose it is possible.. However I did try to emulate the UX of another [similar plugin](https://marketplace.visualstudio.com/items?itemName=WallabyJs.quokka-vscode) (but for Javascript) and I don't believe that's a feature for them either.

If you could get a rally around this feature than I could justify adding it, but every point in the code where the output is required slows it down considerably (the plugin uses a rather naive brute-force method of evaluating the line in question) so I feel pretty comfortable leaving it the way it is for the time being.

Note: Please don't take it personal (it is a very good idea :) ) since I do really appreciate you and others still enjoying Wolf, especially after all these years.

Edit: I re-read your comment and having it be a controlled by a field in settings seems reasonable enough. So yeah, maybe ;)
